samples/NativeAOT/NativeAotTypeManager.cs

```diff
@@ -12,15 +12,875 @@ partial class NativeAotTypeManager : JniRuntime.JniTypeManager {
 
 	// TODO: list of types specific to this application
 	Dictionary<string, Type> typeMappings = new () {
-		["android/app/Activity"]                = typeof (Android.App.Activity),
-		["android/app/Application"]             = typeof (Android.App.Application),
-		["android/content/Context"]             = typeof (Android.Content.Context),
-		["android/content/ContextWrapper"]      = typeof (Android.Content.ContextWrapper),
-		["android/os/BaseBundle"]               = typeof (Android.OS.BaseBundle),
-		["android/os/Bundle"]                   = typeof (Android.OS.Bundle),
-		["android/view/ContextThemeWrapper"]    = typeof (Android.Views.ContextThemeWrapper),
+		// Declared via C# attributes
 		["my/MainActivity"]                     = typeof (MainActivity),
 		["my/MainApplication"]                  = typeof (MainApplication),
+
```

Comment:
This large set of changes makes me want a `NativeAotTypeManager.g.cs` file to contain the `typeMappings` dict initialization, so that everything else is separate.